Rhodes update runtime constants

dobertRowneySr opened the pull request:
updated according to #3537

dobertRowneySr committed:
max inflation set

dobertRowneySr committed:
forum constants set

dobertRowneySr committed:
council budget config set

dobertRowneySr committed:
nft chainspec config ste

dobertRowneySr committed:
updated metadata

mnaamani reviewed (commented):
Changes are correct. However the changes to the NFT config values, `max_starting_price` and `max_bid_step`, only have affect at genesis/new chain. We need to update the implementation of `on_runtime_upgrade()` in `runtime/src/runtime_api.rs` to set the storage values in the content pallet.

dobertRowneySr committed:
NFT constants are updated during migration
    I decided not to use an internal module function like
    `Content::set_nft_values` mainly because I had problem converting
    1billion u64 into `BalanceOf` type used in Content

mnaamani reviewed (approved):
LGTM just waiting on CI checks to complete.
